README.md

```diff
@@ -143,8 +143,8 @@ OTELib is released under the [MIT license](LICENSE) with copyright &copy; SINTEF
 
 OTEAPI Core has been supported by the following projects:
 
-* __OntoTrans__ (2020-2024) that receives funding from the European Union’s Horizon 2020 Research and Innovation Programme, under Grant Agreement no. 862136.
```

Comment:
Any reason to change from __formating__ to **formatting** ?

Reply:
Only to satisfy the markdown-linter I have locally, which wants the formatting to be the same throughout as much as possible. I.e., if `**` is previously used, it should be used throughout. It doesn't prefer one type in favor of another, just consistency. So, if you prefer it one way or another I don't mind using `__` instead of `**` throughout?